### ISSUE TITLE
Retrospectives Teams assessments: When focus moves to the info button next to the question and tooltip automatically appears, the content is not announced, nor can a screen reader user navigate to it to read the content.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vsts-extension-retrospectives#1063]
**Describe the bug**

Repro Steps:
Open an ADO project
Select board > Retrospectives from left navigation bar 
Activate Team assessment button
Turn on Narrator or other screen reader
Tab to the Info "(!)" button that is to the right of the question.  An tooltip opens automatically.
Observe whether the tooltip is announced or if there is a way to read the text there.

Actual Result:
The tooltip is not announced.  
Pressing down arrow cannot move focus to the tooltip to read the content.
There is no indication that a tooltip is shown.

![Image](https://github.com/user-attachments/assets/ab98a036-a3be-48fe-a196-798aa8704c23)

**Comment by msftedad** [comment]
#A11y #accessibility
